src/createReducer.test.ts

```diff
@@ -47,4 +46,57 @@ test('createReducer should create a redux reducer that updates the state on upda
     expect(newState.d).not.toBe(defaultState.d);
     expect(newState.d.e).toBe(3);
     expect(defaultState.d.e).toBe(2);
+});
+
+test('createReducer should create a redux reducer and an UPDATE_ACTION should update the state and a RESET_ACTION reset to default', () => {
+    const reducer = createReducer(defaultState);
+    const nonSupportedActionType = 'NON_SUPPORTED_ACTION';
+    const updateActionType = ACTION_PREFIX + 'UPDATE';
+    const resetActionType = ACTION_PREFIX + 'RESET';
+
+    expect(reducer(defaultState, { type: nonSupportedActionType })).toBe(defaultState);
+
+    expect(reducer(defaultState, {
+        type: updateActionType,
+        meta: { path: 'a' },
+        payload: 'ab',
+    }).a).toBe('ab');
+
+    expect(reducer(defaultState, {
+        type: resetActionType,
+        meta: { path: 'a' },
+        payload: '',
+    }).a).toBe('aa');
```

Comment:
Je geeft de defaultState mee als state aan de reducer. Hoe test je dan of de reset werkt? Als ie niks doet returnt ie ook al het gewenste resultaat. Verzin een state die afwijkt van de defaultState en geef die mee aan de reducer met de reset action. Test dan of in de return value het pad overeenkomt met de defaultState; dan is de reset geslaagd.